job-dsls/jobs/osbs/osbs_build_pipeline.jenkinsfile

```diff
@@ -0,0 +1,230 @@
+@Library(['jenkins-pipeline-shared-libraries','qe-library']) _
+
+Map<String, String> buildResults = [:]
+
+Map<String, String> imageBuildReferences = [:]
+Map<String, String> operatorBuildReferences = [:]
+
+String propertyFileUrl = "${CI_MESSAGE}"
+
+TIMEOUT = 4
+
+pipeline {
+    options {
+        timeout(time: TIMEOUT, unit: 'HOURS')
+    }
+
+    agent {
+        label 'kie-rhel7 && !built-in'
+    }
+
+    environment {
+        PRODUCT_NAME = PROD_COMPONENT.trim()
+        VERSION = PROD_VERSION.trim()
+    }
+
+    stages {
+        stage('Clean workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+
+        stage('Ready CI MESSAGE') {
+            steps {
+                script {
+                    if ( CI_MESSAGE == null || CI_MESSAGE == "" ) {
+                        logConsole.info("CI_MESSAGE is null or empty. Taking property file url from job parameter. ")
+                        propertyFileUrl="${PROPERTY_FILE_URL}"
+                    } else {
+                        logConsole.info("CI_MESSAGE is not null nor empty. Taking property file url from CI-message.")
+                        propertyFileUrl="${CI_MESSAGE}"
```

Comment:
is the CI properly formatted only with the URL?

Reply:
yes it is, in same way is consumed this message in qe testing pipeline